Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -52,6 +62,53 @@ class TabTrayViewController: UIViewController, TabTrayController {
         return shouldUseiPadSetup() ? LegacyTabTrayViewModel.Segment.allCases.map { $0.label } : iPhoneItems
     }
 
+    private lazy var deleteButton: UIBarButtonItem = {
+        return createButtonItem(imageName: StandardImageIdentifiers.Large.delete,
+                                action: #selector(deleteTabsButtonTapped),
+                                a11yId: AccessibilityIdentifiers.TabTray.closeAllTabsButton,
+                                a11yLabel: .AppMenu.Toolbar.TabTrayDeleteMenuButtonAccessibilityLabel)
+    }()
+
+    private lazy var newTabButton: UIBarButtonItem = {
+        return createButtonItem(imageName: StandardImageIdentifiers.Large.plus,
+                                action: #selector(newTabButtonTapped),
+                                a11yId: AccessibilityIdentifiers.TabTray.newTabButton,
+                                a11yLabel: .TabTrayAddTabAccessibilityLabel)
+    }()
+
+    private lazy var flexibleSpace: UIBarButtonItem = {
+        return UIBarButtonItem(barButtonSystemItem: .flexibleSpace,
+                               target: nil,
+                               action: nil)
+    }()
+
+    private lazy var doneButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(barButtonSystemItem: .done,
+                                     target: self,
+                                     action: #selector(doneButtonTapped))
+        button.accessibilityIdentifier = AccessibilityIdentifiers.TabTray.doneButton
+        return button
+    }()
+
+    private lazy var newTabButtonIpad: UIBarButtonItem = {
+        return createButtonItem(imageName: StandardImageIdentifiers.Large.plus,
+                                action: #selector(newTabButtonTapped),
+                                a11yId: AccessibilityIdentifiers.TabTray.newTabButton,
+                                a11yLabel: .TabTrayAddTabAccessibilityLabel)
+    }()
+
+    private lazy var fixedSpace: UIBarButtonItem = {
+        let fixedSpace = UIBarButtonItem(barButtonSystemItem: .fixedSpace,
+                                         target: nil,
+                                         action: nil)
+        fixedSpace.width = CGFloat(UX.NavigationMenu.height)
```

Comment:
This sets width but uses the `UX.NavigationMenu.height` - I am confused 🤔

Reply:
Copy-paste from legacy class, let me check why uses height for width

Reply:
We can maybe rename worst case, if `UX.NavigationMenu.height` is only used for this. I am just trying to avoid having those strange things moved on to the new codebase